src/main/java/duke/commands/ListCommand.java

```diff
@@ -0,0 +1,47 @@
+package duke.commands;
+
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Class that simulates the add command of the user.
+ */
+
+public class ListCommand extends Command {
+    private static final String EMPTY_MSG = "Your list is empty!!!";
+    private static final String SHOW_TASK = "Here are the tasks in your list:";
+    /**
+     * Creates an ListCommand object.
+     * 
+     * @param inputArr Array of length 2 that contains information of the user input
+     *                 At index 0, contains the type of command
+     *                 At index 1, contains the message of the command.
+     */
+    public ListCommand(String[] inputArr) {
+        super(inputArr);
+    }
+
+    public String execute(TaskList tasks, Ui ui, Storage storage) {
+        return showListTasks(tasks, ui);
+    }
+
+    /**
+     * Printing out all the task in the list.
+     * 
+     * @param tasks Object contains the task list.
+     * @param ui Object that deals with interactions with the user.
+     */
```

Comment:
Perhaps you may include @return in JavaDoc.

Reply:
yup thanks for the spot!